### PR TITLE
B 20529 MAIN add save retrieve and filter capability for assigned column sc

### PR DESCRIPTION
**Pull request by joeydoyecaci**
## Summary
Adds in Assigned User Column on Service Counselors Queue with the ability to filter, sort, and assign moves to a Service Counselor.

### How to test as SC
1. Log in as a Service Counselor
2. Click on the dropdown of any move under Assigned, and select a user from the dropdown
3. Repeat step 2 for another user
4. Refresh the page
5. Enter the name of the user that was assigned for either of the moves in the Assigned User column header
6. There should be a pill off to the left of the top of the table with the name of the user filtered and an 'X', only moves that are assigned to the user entered into Assigned User as a filter should display.
7. Click the 'X' button for the filter to remove this filter. After doing so, all moves should show regardless of assigned
8. Click on the Assigned Column to sort in Ascending order, moves should show based on the name of who's assigned
9. Click again to show assigned users in Descending order, moves should show like before but in descending order
10. Click once more to show assigned moves in unordered mode.

### CODE REVIEW

**Comment by joeydoyecaci** [comment]
There is a delta between move_factory in this PR vs main. the change is present in the file but does not show as a change (space between defaultLocator and move).

File: src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx [hunk 1]
```diff
@@ -617,8 +635,6 @@ const ServicesCounselingQueue = ({ userPrivileges, currentUserId }) => {
           csvExportQueueFetcherKey="queueMoves"
           sessionStorageKey={queueType}
           key={queueType}
-          isSupervisor={!!supervisor}
```
Comment (pambecker) [inline src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:620]: Are these 2 lines removed by mistake?  they are not part of your INT PR.

Comment (joeydoyecaci) [reply src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:620]: More likely due to a merge. isSupervisor was removed in a few other areas too as part of it.

Comment (joeydoyecaci) [reply src/pages/Office/ServicesCounselingQueue/ServicesCounselingQueue.jsx:620]: Delta has been removed.

**Review by pambecker — APPROVED** [review]

**Review by paulstonebraker — APPROVED** [review]

**Pull request merged**